docs/ref/applications.txt

```diff
@@ -509,3 +519,31 @@ Here are some common problems that you may encounter during initialization:
   :setting:`INSTALLED_APPS` to contain
   ``'django.contrib.admin.apps.SimpleAdminConfig'`` instead of
   ``'django.contrib.admin'``.
+
+* ``RuntimeWarning: Accessing the database during app initialization is
+  discouraged.`` This happens if one of your apps accesses the databases in the
```

Comment:
```suggestion
  discouraged.`` This happens if one of your apps attempts database access in the
```
I was going to suggest removing `s` from `databases` but `attempts datbase access` can be used for 1 or many dbs.

Reply:
I'd prefer David's version with `database access` instead of `SQL queries` as it's closer to the raised exception and more generic.